JS/Mery/Health-App/src/examples/FitnessAppScreen.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Icon from '@mdi/react'
+import {
+  mdiHome,
+  mdiBell,
+  mdiFood,
+  mdiAccount,
+  mdiChartLine,
+} from '@mdi/js'
+import {
+  H4,
+  Card,
+  withResize,
+  ToggleButton,
+} from 'ui-neumorphism'
+
+class FitnessAppScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.getIconProps = this.getIconProps.bind(this);
+  }
+
+  getIconProps(index) {
+    return {
+      size: this.props.selectedIndex === index ? 0.9 : 0.8,
```

Comment:
Can you explain what magical numbers are those

Reply:
when the icon is selected the icon`s size has to be 0.9  else 0.8

Reply:
I understand that part but can you explain why those magical numbers

Reply:
I have tried other sizes , for me this  sizes are looking good in my tabs.